neuro_san/http_sidecar/handlers/connectivity_handler.py

```diff
@@ -27,13 +27,17 @@ async def get(self, agent_name: str):
         """
         Implementation of GET request handler for "connectivity" API call.
         """
+        metadata: Dict[str, Any] = self.get_metadata()
+        update_done: bool = await self.update_agents(metadata)
+        if not update_done:
+            return
+
```

Comment:
Here and in two other handlers below:
before starting to execute this request, query the "host" side for current list of known agents,
before checking if we should route this request at all.

Reply:
Why would we not try to service the request at all simply because the update was not done?
Maybe worth a comment in the if/return clause above.

Reply:
If this kind of update is not done, that means something is broken real bad inside our code.
In this case, we call "process_exception()" helper method inside and finish request as 500 "Internal server error".